src/components/themed/TransactionListRow.tsx

```diff
@@ -150,21 +150,23 @@ export function TransactionListRow(props: Props) {
 
   // Pending Text and Style
   const currentConfirmations = transaction.confirmations
-  const isConfirmed = currentConfirmations === 'confirmed'
 
-  const unconfirmedOrTimeText = isConfirmed
-    ? unixToLocaleDateTime(transaction.date).time
-    : !isSentTransaction && canReplaceByFee && currentConfirmations === 'unconfirmed'
-    ? lstrings.fragment_transaction_list_unconfirmed_rbf
-    : currentConfirmations === 'unconfirmed'
-    ? lstrings.fragment_wallet_unconfirmed
-    : currentConfirmations === 'dropped'
-    ? lstrings.fragment_transaction_list_tx_dropped
-    : typeof currentConfirmations === 'number'
-    ? sprintf(lstrings.fragment_transaction_list_confirmation_progress, currentConfirmations, requiredConfirmations)
-    : lstrings.fragment_transaction_list_tx_synchronizing
+  const unconfirmedOrTimeText =
+    currentConfirmations === 'confirmed'
+      ? unixToLocaleDateTime(transaction.date).time
+      : !isSentTransaction && canReplaceByFee && currentConfirmations === 'unconfirmed'
+      ? lstrings.fragment_transaction_list_unconfirmed_rbf
+      : currentConfirmations === 'unconfirmed'
+      ? lstrings.fragment_wallet_unconfirmed
+      : currentConfirmations === 'dropped'
+      ? lstrings.fragment_transaction_list_tx_dropped
+      : currentConfirmations === 'failed'
+      ? lstrings.fragment_transaction_list_tx_failed
+      : typeof currentConfirmations === 'number'
+      ? sprintf(lstrings.fragment_transaction_list_confirmation_progress, currentConfirmations, requiredConfirmations)
+      : lstrings.fragment_transaction_list_tx_synchronizing
 
-  const unconfirmedOrTimeStyle = isConfirmed ? styles.secondaryText : styles.unconfirmedText
+  const confirmationStyle = currentConfirmations === 'failed' ? styles.failedText : styles.unconfirmedText
```

Comment:
With this change, wouldn't confirmed transactions appear in yellow `warningText` style now? 

I think we would want `isConfirmed ? null : currentConfirmations === 'failed' ? styles.failedText : styles.unconfirmedText`, so we don't apply any color tweaks to the fully-confirmed case.